dspace-rest/src/main/java/org/dspace/rest/filter/ItemFilterUtil.java

```diff
@@ -0,0 +1,263 @@
+        		names.add(bit.getName());
+        	}
+        }
+		return names;
+	}
+
+	
+	static int countOriginalBitstreamMime(Context context, Item item, String mimeList) {
+		return countBitstreamMime(context, BundleName.ORIGINAL, item, mimeList);
+	}
+	static int countBitstreamMime(Context context, BundleName bundleName, Item item, String mimeList) {
+		int count = 0;
+        for(Bundle bundle: item.getBundles()){
+        	if (!bundle.getName().equals(bundleName.name())){
+        		continue;
+        	}
+        	for(Bitstream bit: bundle.getBitstreams()) {
+            	for(String mime: mimeList.split(",")) {
+                    try {
+						if (bit.getFormat(context).getMIMEType().equals(mime)) {
+							count++;
+						}
+					} catch (SQLException e) {
+						log.error("Get format error for bitstream " + bit.getName());
+					}
+            	}
+            }
+        }
+		return count;
+	}
+
+	static int countBitstreamByDesc(BundleName bundleName, Item item, String descList) {
+		int count = 0;
+        for(Bundle bundle: item.getBundles()){
+        	if (!bundle.getName().equals(bundleName.name())){
+        		continue;
+        	}
+        	for(Bitstream bit: bundle.getBitstreams()) {
+            	for(String desc: descList.split(",")) {
+            		String bitDesc = bit.getDescription();
+            		if (bitDesc == null) {
+            			continue;
+            		}
+                    if (bitDesc.equals(desc)) {
+                    	count++;
+                    }
+            	}
+            }
+        }
+		return count;
+	}
+
+	static int countBitstreamSmallerThanMinSize(Context context, BundleName bundleName, Item item, String mimeList, String prop) {
+		long size = ConfigurationManager.getLongProperty("rest", prop);
+		int count = 0;
+        try {
+            for(Bundle bundle: item.getBundles()){
+            	if (!bundle.getName().equals(bundleName.name())){
+            		continue;
+            	}
+            	for(Bitstream bit: bundle.getBitstreams()) {
+                	for(String mime: mimeList.split(",")) {
+                        if (bit.getFormat(context).getMIMEType().equals(mime)) {
+                        	if (bit.getSize() < size) {
+                            	count++;                        		
+                        	}
+                        }
+                	}
+                }
+            }
+        } catch (SQLException e) {
+        }
+		return count;
+	}
+	
+	static int countBitstreamLargerThanMaxSize(Context context, BundleName bundleName, Item item, String mimeList, String prop) {
+		long size = ConfigurationManager.getLongProperty("rest", prop);
+		int count = 0;
+        try {
+            for(Bundle bundle: item.getBundles()){
+            	if (!bundle.getName().equals(bundleName.name())){
+            		continue;
+            	}
+            	for(Bitstream bit: bundle.getBitstreams()) {
+                	for(String mime: mimeList.split(",")) {
```

Comment:
Perhaps also trim the mime ? Because when an space occurs between the commas the code will not work properly.